### ISSUE TITLE
Image block unaware of edits to image

### ISSUE THREAD

**Issue by rogerlos** [WordPress/gutenberg#16977]
If you drag an image into the image block while editing a post and then edit the image (to, say, rotate it), Gutenberg is unaware of the edited changes if you use the block controls to switch to a different image size after the edit.

1. Add an image to your post by using the image block and dragging an image to it.
2. Use the pencil icon on the block to edit the image, and then within the media modal choose "edit image".
3. Rotate the image, save, and update the media.
4. Return to edit your post, and in the image block inspector, change the image size.

What you will see in the editor once you change the size is the image as it was before it was edited. Exiting the post editor and re-entering it will show you the correct image.

**Comment by rogerlos** [comment]
Post preview also seems unaware of the edits, FWIW.

**Comment by youknowriad** [comment]
Isn't this a browser cache issue? Also, I think it's probably related to server-side code, so creating a trac ticket might be better here.

**Comment by paaljoachim** [comment]
Testing:

Using WordPress 5.6.1
Theme: Twenty Twenty One
No plugins activated.
Chrome
A local test site that I use.
OSX 10.15.7

Testing instructions:
1. Add an image to your post by using the image block and dragging an image to it.
2. Use the pencil icon on the block to edit the image, and then within the media modal choose "edit image".
3. Rotate the image, save, and update the media.
4. Return to edit your post, and in the image block inspector, change the image size.

Added an Image block. Dragged an image into it.
Clicked the Replace button. Within the media modal. I clicked the Edit button.
Clicked the Crop tool. Rotated image. Clicked Apply. 
Saved draft.
Exited screen and re entered. Noticed that the image had reset itself.

-----------

Same test with beta 1 of WordPress 5.7
(WordPress Beta Tester plugin is activated.)

This time the image showed as rotated.

Conclusion for WordPress 5.7 this issue has been fixed.
I will go ahead and close it.
